app/reducers/events.js

```diff
@@ -68,14 +68,16 @@ function mutateEvent(state: any, action: any) {
           [eventId]: {
             ...stateEvent,
             loading: false,
-            registrationCount: stateEvent.registrationCount + 1,
+            registrationCount: registration.pool
+              ? stateEvent.registrationCount + 1
+              : stateEvent.registrationCount,
```

Comment:
why would the pool not exist if registrationCount exists?

Reply:
When it is a socket registration to waiting list – the pool is not set.

Reply:
alright